### PR TITLE
[python] Test_UserLoginSuccessEvent fails on 3.12

### PR DESCRIPTION
**Pull request by avara1986**
## Motivation
Those PRs are failing due to this error:
https://github.com/DataDog/system-tests/pull/2434
https://github.com/DataDog/system-tests/pull/2433
## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [x] [Relevant label](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md) (`run-parametric-scenario`, `run-profiling-scenario`...) are presents
* [x] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [x] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [x] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: manifests/python.yml [hunk 1]
```diff
@@ -413,8 +413,12 @@ tests/:
       Test_NoLimitOnWafRules: v1.1.0rc2.dev
     test_event_tracking.py:
       Test_CustomEvent: v1.10.0
-      Test_UserLoginFailureEvent: v1.9.0
-      Test_UserLoginSuccessEvent: v1.9.0
+      Test_UserLoginFailureEvent:
+        '*': v1.9.0
+        python3.12: bug (stars v2.8.0)
```
Comment (robertomonteromiguel) [inline manifests/python.yml:418]: typo
could we add more information about the bug? Reference to PR, to Jira.....

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
typo

File: manifests/python.yml (hunk 1 above)
Comment (avara1986) [reply manifests/python.yml]: @robertomonteromiguel : https://datadoghq.atlassian.net/browse/APPSEC-53079 :+1:

**Review by juanjux — APPROVED** [review]

**Review by robertomonteromiguel — APPROVED** [review]

File: manifests/python.yml [hunk 2]
```diff
@@ -413,8 +413,12 @@ tests/:
...
     test_event_tracking.py:
       Test_CustomEvent: v1.10.0
-      Test_UserLoginFailureEvent: v1.9.0
-      Test_UserLoginSuccessEvent: v1.9.0
+      Test_UserLoginFailureEvent:
+        '*': v1.9.0
+        python3.12: bug (APPSEC-53079)
+      Test_UserLoginSuccessEvent:
+        '*': v1.9.0
+        python3.12: bug (stars v2.8.0)
```
Comment (gnufede) [inline manifests/python.yml:421]:
```suggestion
        python3.12: bug (starts from v2.8.0)
```

Not sure what do you mean by stars

**Pull request merged**